{"uuid": "1f46a987-30fb-4c6f-8a44-1619d1e9153c", "children": ["5e1ecf64-94d4-416d-9b85-54c70c6ed518", "402174fb-4257-40fe-a363-14dc2a5e3fff", "1817548d-27a1-4b61-928f-988082535df5", "f54cacbf-6aee-4f5d-af4b-376504862ec3", "08f11575-4085-42d8-979e-2e771556f2e5", "331d3981-553d-4fa5-920e-4b1eb4841de6", "93537f95-07c0-4cec-a21a-71c93c35e54b", "98e246a8-9de1-45c3-82e0-6026743035c4", "30c59ba4-c3a9-4e8a-842f-d462c4c6779b"], "befores": [{"name": "get_job_details", "status": "broken", "statusDetails": {"message": "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy-1.5.0-py3.10.egg/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/Test_API_Suite/test_corteva.py\", line 16, in get_job_details\n    peviitor_jobs_data = testutils.scrape_peviitor(company_name, 'Rom√¢nia')\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/utils.py\", line 70, in scrape_peviitor\n    response_data = TestUtils._get_request(params)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/utils.py\", line 48, in _get_request\n    updateapi.get_token()\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/sites/__utils/peviitor_update.py\", line 42, in get_token\n    self.access_token = requests.request(\"POST\", \"https://api.peviitor.ro/v5/get_token/\", headers=post_header, data=payload).json()['access']\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/models.py\", line 975, in json\n    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n"}, "start": 1714108493538, "stop": 1714108497819}], "start": 1714108493538, "stop": 1714108498183}